site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,128 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
+
+### Setting up your Mattermost instance and AWS for deploying custom apps
```

Comment:
It's not only about customs apps. On-prem customers can also use this method to deploy Mattermost written ones like Zendesk.

Reply:
@hanzei, that is true.  I think the overall feel of the docs should be that an App is understood at this point and we can remove the `custom` and 3rd party references.  An app is just an "App" that is already defined.

Reply:
I removed references to `custom apps` and replaced with just `apps`.

[`32e4489` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/32e4489e837bfeb88d2e0e599aebc22a438813ee)